core/src/main/scala/kafka/server/AutoTopicCreationManager.scala

```diff
@@ -138,6 +138,19 @@ class DefaultAutoTopicCreationManager(
         } else if (response.versionMismatch() != null) {
           warn(s"Auto topic creation failed for ${creatableTopics.keys} with invalid version exception")
         } else {
+          if (response.hasResponse) {
+            response.responseBody() match {
+              case createTopicsResponse: CreateTopicsResponse =>
+                createTopicsResponse.data().topics().forEach(topicResult => {
+                  val error = Errors.forCode(topicResult.errorCode())
+                  if (error != Errors.NONE) {
+                    warn(s"Auto topic creation failed for ${topicResult.name()} with error '${error.name()}': ${topicResult.errorMessage()}")
```

Comment:
tiny nit: we can skip `()` for getters
```suggestion
                    warn(s"Auto topic creation failed for ${topicResult.name} with error '${error.name}': ${topicResult.errorMessage}")
```

Reply:
thanks, have killed off `()`